RockLib.Messaging.Http/HttpClientSender.cs

```diff
@@ -77,36 +136,52 @@ public void Dispose()
         /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         public async Task SendAsync(SenderMessage message, CancellationToken cancellationToken)
         {
-            if (message.OriginatingSystem == null)
+            if(message is null)
+            {
+                throw new ArgumentNullException(nameof(message));
+            }
+            if (message.OriginatingSystem is null)
+            {
                 message.OriginatingSystem = "HTTP";
+            }
```

Comment:
This is confusing. If `message` is `null`, then won't trying to set `OriginatingSystem` throw a `NullReferenceException`?

Now, true, you put in a `null` check, but then `message` should **never** be `null`.

Reply:
@JasonBock So VS was giving a build error that I needed to validate `message` is non-null before using it.  Is there a better way to handle this here?